src/locales/en/translations.json

```diff
@@ -90,6 +90,22 @@
     "suggestions could not be retrieved": "",
```

Comment:
I thought we only need to push the `.pot` files? Am I confused?

Reply:
This will be removed as part of the [App platform PR](https://github.com/dhis2/capture-app/pull/876)